big_tests/tests/connect_SUITE.erl

```diff
@@ -148,32 +148,33 @@ end_per_suite(Config) ->
     escalus:end_per_suite(Config).
 
 init_per_group(c2s_noproc, Config) ->
-    config_ejabberd_node_c2s(Config, [starttls]),
+    configure_c2s_listener(Config, #{tls => [starttls | common_tls_opts(Config)]}),
     Config;
 init_per_group(session_replacement, Config) ->
-    config_ejabberd_node_c2s(Config, [starttls]),
+    configure_c2s_listener(Config, #{tls => [starttls | common_tls_opts(Config)]}),
     logger_ct_backend:start(),
     Config;
 init_per_group(starttls, Config) ->
-    config_ejabberd_node_c2s(Config, [starttls_required]),
+    configure_c2s_listener(Config, #{tls => [starttls_required | common_tls_opts(Config)]}),
     Config;
 init_per_group(tls, Config) ->
-    config_ejabberd_node_c2s(Config, [tls]),
+    configure_c2s_listener(Config, #{tls => [tls | common_tls_opts(Config)]}),
     Users = proplists:get_value(escalus_users, Config, []),
     JoeSpec = lists:keydelete(starttls, 1, proplists:get_value(?SECURE_USER, Users)),
     JoeSpec2 = {?SECURE_USER, lists:keystore(ssl, 1, JoeSpec, {ssl, true})},
     NewUsers = lists:keystore(?SECURE_USER, 1, Users, JoeSpec2),
     Config2 = lists:keystore(escalus_users, 1, Config, {escalus_users, NewUsers}),
     [{c2s_port, ct:get_config({hosts, mim, c2s_port})} | Config2];
 init_per_group(feature_order, Config) ->
-    config_ejabberd_node_c2s(Config, [starttls_required, {zlib, 10000}]),
+    configure_c2s_listener(Config, #{zlib => 100000,
```

Comment:
I've just noticed it, but this zero makes zero difference (sic!) in test results ;-)